Jenkinsfile

```diff
@@ -44,12 +44,16 @@ def deployBootloaderRepoStep() {
         dir(repoName) {
           deleteDir()
           checkout scm
-          execute("mbed deploy --protocol ssh")
-          execute("mbed ls")
-          execute("mkdir release")
-          execute("python ./scripts/make_release.py --patch -o release")
-          execute("cd mbed-os;git archive -o ../mbed.tar HEAD")
-          execute("tar cjf mbed-bootloader.tar.bz2 --exclude mbed-os --exclude mbed-bootloader.tar.bz2 * .[a-z]*")
+          sh ('''
+            mbed deploy --protocol ssh
+            mbed ls
+            mkdir release
+            python ./scripts/make_release.py --patch -o release
```

Comment:
python2 maybe.

Reply:
Lets see what the machine tells..

I removed the `python2` from the other section as well. We should prepare for Python3 already.